### PR TITLE
Fix #128: clarify effect of setting the value attribute with respect …

### LINKED ISSUE #128: Note that setting AudioParam.value will be ignored when any automation events have been set on the object
> Originally reported on W3C Bugzilla [ISSUE-19765](https://www.w3.org/Bugs/Public/show_bug.cgi?id=19765) Mon, 29 Oct 2012 23:07:00 GMT
> Reported by Ehsan Akhgari [:ehsan]
> Assigned to 

We should note that if the author sets the value property on an AudioParam which has any automation events scheduled, the new value will be silently ignored without any exception being raised.

### PR DESCRIPTION
**Pull request by joeberkovitz**
…to equivalent parameter automation action.

### CODE REVIEW

**Comment by rtoy** [comment]
While we agreed to this a while back, during our uncall today, @cwilso, @hoch, and @rtoy coincidentally chatted (off-line) about this.  I think we need to discuss the implications with everyone.  Does equivalent mean actually doing everything that setValueAtTime would do, including adding an event into the timeline?

**Comment by joeberkovitz** [comment]
Thanks for sharing with the class :-)  How about I reword this to indicate that it is identical to setValueAtTime in terms of value changes, but there is no actual automation event in the timeline (which would drag in a bunch of other baggage including, strangely, disabling subsequent assignments to the setter)?

**Comment by joeberkovitz** [comment]
Oh now I see your comments in #503 which sounds like you're headed the other way: you want it to be equivalent (and in turn to have the setter no longer cancel automations). Yeah, we should discuss. I like that idea too, at least right now.
